omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/VisitAttributeController1_9Test.java

```diff
@@ -109,4 +110,14 @@ public void shouldVoidAVisitAttribute() throws Exception {
 		Assert.assertTrue(visitAttribute.isVoided());
 		Assert.assertEquals("unit test", visitAttribute.getVoidReason());
 	}
+	
+	@Test
+	public void shouldNotSaveVisitAttributeIfMaxOccursIsNull() {
+		VisitAttribute visitAttribute = service.getVisitAttributeByUuid("7c2tyr18-6faa-11e0-7899-001e378eb66d");
```

Comment:
Are you sure that the REST call will not save if max occurs is null?

Reply:
I have corrected it and I guess it makes sense now.